Task1.cs

```diff
@@ -0,0 +1,67 @@
+﻿
```

Comment:
@noor848  Could you re-solve this problem without using Dictionary? think on that, Don't update the PR, will discussed it.